zuul.d/projects.yaml

```diff
@@ -6,3 +6,6 @@
     github-gate:
       jobs:
         - noop
+    github-experimental:
+      jobs:
+        - create-scenario010-trigger-for-downstream-network-zed-c9-rdo-test
```

Comment:
I would first start with a noop job. We can add the trigger job in a seperate pr.

Reply:
done